bindings/c/test/mako/mako.cpp

```diff
@@ -76,8 +78,20 @@ using namespace mako;
 
 thread_local Logger logr = Logger(MainProcess{}, VERBOSE_DEFAULT);
 
+Transaction createNewTransaction(Database* db, Arguments const& args, int id = -1) {
+	// No tenants specified
+	if (args.tenants <= 0) {
+		return db->createTransaction();
+	}
+	// Create Tenant Transaction
+	int tenant_id = (id == -1) ? urand(1, args.tenants) : id;
```

Comment:
For now, using random selection, but logic here will need to be more sophisticated for our testing goals